partner_address_street3/tests/test_street_3.py

```diff
@@ -1,88 +1,45 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Author: Nicolas Bessi
-#    Copyright 2014 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2014 Nicolas Bessi, Alexandre Fayolle, Camptocamp SA
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
 import openerp.tests.common as test_common
+from ..hooks import uninstall_hook
 
 
 class TestStreet3(test_common.TransactionCase):
 
     def test_partner(self):
-        part_model = self.registry('res.partner')
-        country_model = self.registry('res.country')
-        country_id = country_model.create(
-            self.cr,
-            self.uid,
-            {
-                'name': 'Donut Land',
-                'code': 'DNL',
-            }
-        )
+        # Test address_format has been updated on existing countries
```

Comment:
Use a correct docstring for the method. This way, the docstring appears in the logs too.